### DISCUSSION TITLE
Change password?

### DISCUSSION THREAD

**Discussion by Avpman2** [dahlb/ha_carrier#144] [Q&A]
I changed my Carrier account password but I can't figure out where to change it in the HA configuration. Anyone know where?
TIA!

**Comment by dahlb** [comment]
I think in the current ui you add hub and put in the same username with the new password and it should replace the existing one with the updated password.
<img width="1086" alt="Screenshot 2024-09-05 at 10 32 48 AM" src="https://github.com/user-attachments/assets/6d34d683-4d41-4b36-8d77-92ff3567209d">

**Comment by Avpman2** [comment]
I already tried that, and it creates a new instance (hub) with no entities. There has to be some way to change it. No?

**Reply by maxbadrak** [reply]
I know very little about HA, so ended up deleting the entire integration, redownloading it again, banging head against wall, since it didn't show up in the list of integrations, then finally went to `/config/integrations/integration/ha_carrier`, and was able to add the hub there, and it's working now.
